ochre/Equipment/HVAC.py

```diff
@@ -123,8 +123,11 @@ def __init__(self, envelope_model=None, use_ideal_capacity=None, **kwargs):
         self.duct_dse = ducts.get('DSE (-)')  # Duct distribution system efficiency
         self.duct_zone = self.envelope_model.zones.get(ducts.get('Zone'))
         if self.duct_dse is None:
-            # Calculate DSE using ASHRAE 152
-            self.duct_dse = utils_equipment.calculate_duct_dse(self, ducts, **kwargs)
+            if self.name == 'Room AC':
```

Comment:
I think we could add some more exceptions here. If ducts are in a conditioned basement, I think we can skip the 152 calculation too, right @jmaguire1? Any other things we should catch based on the bad DSE runs?